extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/OpenTelemetryRecorder.java

```diff
@@ -30,23 +32,23 @@ public class OpenTelemetryRecorder {
 
     public static final String OPEN_TELEMETRY_DRIVER = "io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver";
 
-    /* STATIC INIT */
+    @StaticInit
```

Comment:
I'm all for these marker annotations as I agree things have been confusing. But shouldn't have a check then?

Reply:
> But shouldn't have a check then?

I am not sure I follow this one. Care to elaborate please?

Reply:
If we add a marker, I would expect us to check that this recorder method is called at the proper phase and not at runtime init (for this particular example).